dptb/tests/test_pardisoeig.py

```
@@ -0,0 +1,73 @@
+        edge_field=AtomicDataDict.EDGE_FEATURES_KEY,
+        node_field=AtomicDataDict.NODE_FEATURES_KEY,
+        overlap=False,
+        dtype=torch.float32, 
+        device=torch.device("cpu")
+    )
+
+    adata = model(adata)
+    image_h = hr2hr(adata)
+    image_s = sr2sr(adata)
+
+    h = image_h.sample_k([0,0,0], symm=True).to_scipy(format="csr")
```

Ruff (0.15.0)

[error] Local variable `h` is assigned to but never used

Remove assignment to unused variable `h`

(F841)